1.preprocess_data/scripts/1.extract_metadata.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import pandas as pd
+
+# In[2]:
+
+
+path_to_data = pathlib.Path("../../data/images").resolve(strict=True)
+
+
+# In[3]:
+
+
+# get all files in the data directory
+files = path_to_data.glob("*")
+
+
+# In[4]:
+
+
+metadata_dict = {
+    "condition": [],
+    "cell_number": [],
+    "mitochondria_number": [],
+    "file_type": [],
+    "file_name": [],
+    "file_path": [],
+}
```

Comment:
More naming comments!

- it seems that the entries under the `condition` column are "images". Is there a better name for this column? "images" isn't a condition.
- the `cell_number` column isn't in fact an integer, but a string that says "cell". Is there a better column name that wouldn't confuse?
- the `mitochondria_number` column looks like it is a string as well. Perhaps `mitochondria_id` is better? Depending on how you see yourself (or someone else) interacting with this column, you might also consider adding a character to each entry (e.g., `M1_004`) so that the column can no longer be coerced to a number (assuming the three digit style is helpful)/
- The `file_type` column is not a type entry but a four digit string. I have the same comment about how someone might use this column, preventing int coercion, and updating the name.
- file_name and file_path are clear!

Reply:
I updated each of these to be more clear and concise